src/wallets/hardware/trezor/index.js

```diff
@@ -38,7 +39,9 @@ class TrezorWallet {
   getAccount(idx) {
     const derivedKey = this.hdKey.derive('m/' + idx);
     const txSigner = async tx => {
-      tx = new Transaction(tx, {
+      const locTx = Object.assign({}, tx);
+      locTx['chainId'] = new BigNumber(tx['chainId']).toNumber();
```

Comment:
why is this necessary?

Reply:
Sentry error that expects `chainId` to be an Integer.

Reply:
Reverted